soil/templates/soil/partials/dl_status.html

```diff
@@ -1,4 +1,5 @@
 {% load i18n %}
+{% load hq_shared_tags %}
```

Comment:
thought about it some more. the whole dropbox thing makes it pretty dependent on HQ. it definitely breaks the abstraction